src/internal/translations/de-DE.json

```diff
@@ -63,8 +69,8 @@
     "dashboard.bucket.measures_title.table": "Messungen",
     "dashboard.bucket.attribute_title.table": "Zeilen",
     "dashboard.bucket.columns_title.table": "Spalten",
-    "dashboard.error.missing_primary_bucket_item.heading": "Kein primäres Maß in Ihrer Betrachtung",
-    "dashboard.error.missing_primary_bucket_item.text": "Fügen Sie Ihrer Ansicht ein primäres Maß hinzu oder schalten Sie auf Tabelle um.\nWenn Sie fertig sind, können Sie es speichern.",
+    "dashboard.headline.error.missing_bucket_item.heading": "Kein primäres Maß in Ihrer Betrachtung",
```

Comment:
This is weird. If someone renamed the key, he should rename it also in translations. Please find out who renamed the key and why the did not do it also in translations. This should not happen - author should know about the risks of this approach.

Reply:
But if it is renamed in en-US, we have to merge it asap...

Reply:
Done.